agent/config/config.go

```diff
@@ -393,6 +395,8 @@ func environmentConfig() (Config, error) {
 		CNIPluginsPath:                   cniPluginsPath,
 		AWSVPCBlockInstanceMetdata:       awsVPCBlockInstanceMetadata,
 		AWSVPCAdditionalLocalRoutes:      additionalLocalRoutes,
+		ContainerMetadataEnabled:         containerMetadataEnabled,
```

Comment:
Can you also update `DefaultConfig` to set `ContainerMetadataEnabled` to `false`?